package-lock.json

```diff
@@ -1,8 +1,11641 @@
 {
     "name": "PrairieLearn",
     "version": "3.2.0",
-    "lockfileVersion": 1,
+    "lockfileVersion": 2,
     "requires": true,
+    "packages": {
```

Comment:
If there are no changes to the `package.json`, there shouldn't be any changes to the `package-lock.json`, right?

Reply:
yeah, I dont know why `package-lock.json` keeps changing either

Reply:
certain `npm` commands will change it as a way of keeping track of all the different versions of dependencies you have used in the past (perhaps among some other things, I'm not sure). If you haven't actually changed `package.json`, don't commit `package-lock.json`, keep it as it is. Now that you have committed it, you will need to revert it.

Reply:
reverted